tests/unit/iothub/aio/test_async_clients.py

```diff
@@ -693,6 +694,10 @@ async def test_raises_error_when_message_size_greater_than_256(self, client, mqt
         assert "256 KB" in e_info.value.args[0]
         assert mqtt_pipeline.send_message.call_count == 0
 
+    @pytest.mark.skipif(
```

Comment:
What are these python 3.12 issues? Does this impact the customer at all? Looks like if they have large messages they might not get an error?

Reply:
Yeah - but honestly, this max size never actually worked in practice, so it's not an issue.